Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    public Rigidbody2D body;
+    public SpriteRenderer sprite;
+    public Animator animator;
+    public float speed = 8.0f;
+
+    Vector3 moveDirection = Vector3.zero;
+    string lastAnimation = "";
+
+
+    void Start() 
+    {
+        SetAnimation("idle_down");
+    }
+
+    void Update()
+    {
+        UpdateAnimator();
+    }
+
+    void FixedUpdate()
+    {
+        moveDirection = new Vector3(
+            Input.GetAxisRaw("Horizontal"), 
+            Input.GetAxisRaw("Vertical"), 
+            0.0f
+        );
+        moveDirection.Normalize();
```

Comment:
Normalizing the vector will cause a loss of information if youre using GetAxis. If you want to keep it smooth, dont do it.

Reply:
Yeah, with raw input, you are forced to normalize otherwise diagonal movement are faster